app/src/main/java/com/duckduckgo/app/privacymonitor/ui/PrivacyMonitorGradeExtension.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2017 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.privacymonitor.ui
+
+import com.duckduckgo.app.privacymonitor.HttpsStatus
+import com.duckduckgo.app.privacymonitor.PrivacyMonitor
+import com.duckduckgo.app.privacymonitor.model.PrivacyGrade
+import com.duckduckgo.app.privacymonitor.model.TermsOfService
+
+
+val PrivacyMonitor.score: Int
+    get() {
+        var score = baseScore
+        score += Math.ceil(trackerCount / 10.0).toInt()
+        if (majorNetworkCount > 0) {
```

Comment:
iOS doesn't do this, however iOS does check to see if this domain is part of a major network and adds a point for that.  Could be iOS is missing this and Android is missing the "in major network" test.  Will check iOS against extensions again. :)

Reply:
Yep, extensions doesn't appear to do this, but it does do the "in a" major network check:

https://github.com/duckduckgo/chrome-zeroclickinfo/blob/beta/js/site.js#L111

Reply:
This is equivalent to `inMajorTrackingNetwork`, I just found this clearer as the way extensions have it worded it sounds like a reference to the site when it is really a reference to trackers being in a major network.

Reply:
Yeah, iOS is actually wrong here. We've been checking the current domain is part of a major tracker. Will create a separate PR to fix that in iOS.